packages/mongo/mongo.d.ts

```diff
@@ -18,22 +18,12 @@ export namespace Mongo {
 
   type Query<T> = MongoNpmModule.Filter<T>;
 
-  export type QueryWithModifiers<T> = {
-    $query: Query<T>;
-    $comment?: string | undefined;
-    $explain?: any;
-    $hint?: Hint;
+  export type QueryWithModifiers = {
     $maxScan?: any;
-    $max?: any;
-    $maxTimeMS?: any;
-    $min?: any;
-    $orderby?: any;
-    $returnKey?: any;
-    $showDiskLoc?: any;
     $natural?: any;
```

Comment:
Interestingly, the official driver no longer has typings for any these. Maybe we should remove it too?

Reply:
Yeah, I meant exactly that. But now we have an absurd situation, where we have a `Query` type that's an alias for driver's `Filter` and the only place we use it is our `Selector` type 😛 

I'm not sure what we should do exactly, but I think that staying as close to the driver's types and naming would be beneficial.

Reply:
Agreed! thanks for the insights :)

Reply:
I hope now looks good!